pkg/install/billing.go

```diff
@@ -7,12 +7,15 @@ import (
 	"context"
 	"net/http"
 
+	"github.com/Azure/go-autorest/autorest/azure"
+
 	"github.com/Azure/ARO-RP/pkg/api"
 	"github.com/Azure/ARO-RP/pkg/database/cosmosdb"
+	"github.com/Azure/ARO-RP/pkg/util/billing"
 )
 
 func (i *Installer) createBillingRecord(ctx context.Context) error {
```

Comment:
similarly, I think all of the body of this function would be better in pkg/util/billing.

Reply:
following the pattern used for pkg/util/dns, or pkg/util/privateendpoint

Reply:
I switched to pull e2e specific logic into a function in pkg/util/billing.  Let me know if you want me to move more

Reply:
ideally, yes, if we can encapsulate all the billing pieces in there (i.e. the relevant functionality in delete and create), it will make unit testing easier for everyone

Reply:
Moved them all into utils/billing